order_risk_test.go

```diff
@@ -0,0 +1,339 @@
+package goshopify
+
+import (
+	"errors"
+	"fmt"
+	"github.com/jarcoal/httpmock"
```

Comment:
Can you separate the imports as per the usual convention? stdlib first, then 3rd party imports?